.pre-commit-config.yaml

```diff
@@ -1,14 +1,8 @@
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
 repos:
-
--   repo: local
+-   repo: https://github.com/psf/black
+    rev: 23.9.1
     hooks:
     -   id: black
-        name: black
-        entry: bash -c 'pipenv run black $@' --
-        language: system
-        types: [python]
-        args: []
         exclude: ^(.+)/migrations/.*\.py
-        
```

Comment:
nit: better to end the file with one empty line
[PEP8 "No newline at end of file"](https://stackoverflow.com/questions/71976465/pep8-no-newline-at-end-of-file)